v2/samples/containerservice/v1beta/v1beta20230101_managedcluster.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: containerservice.azure.com/v1beta20230101
```

Comment:
Consider moving different versions into separate dirs as we discussed earlier for `dbforpostgresql`

https://github.com/Azure/azure-service-operator/tree/main/v2/samples/dbforpostgresql

Reply:
Done